dali/test/python/test_RN50_external_source_parallel_utils.py

```diff
@@ -225,6 +284,13 @@ def parse_test_arguments(supports_distributed):
     parser.add_argument(
         "--test_pipes", nargs="+", default=["parallel", "file_reader", "scalar"],
         help="Pipelines to be tested, allowed values: 'parallel', 'file_reader', 'scalar'")
+    parser.add_argument('--source_mode', default="sample", choices=['sample', 'batch', 'generator'], type=str,
+                        help='Available modes: sample, batch, generator. First two run stateless '
+                        'callbacks that return sample or batch given the index, the '
+                        'generator mode iterates over a generator. '
+                        'Parameter value has no effect on file reader pipeline.')
+    parser.add_argument('--mixed_decode', default=False, type=bool,
+                        help='If True decodes with mixed decoder, otherwise decodes on cpu and moves with tensor.gpu()')
```

Comment:
```suggestion
                        help='If True decodes images with DALI's mixed decoder, otherwise decodes on cpu inside external source callback and moves with tensor.gpu()')
```